review_practice/api-server/resolvers/room.js

```diff
@@ -0,0 +1,45 @@
+import { AuthenticationError, UserInputError } from 'apollo-server';
+
+// Validator
+const isAdultValueLessEqualThan0 = searchOptions =>
+    searchOptions && searchOptions.personnel && searchOptions.personnel.adult <= 0;
```

Comment:
옵셔널 체이닝을 이용하면 조금 더 깔끔하겠네요 :)
```suggestion
  searchOptions?.personnel?.adult <= 0;
```